test.js

```diff
@@ -745,8 +745,8 @@ test('drive.download(filename, [options])', async (t) => {
   }
 })
 
-test.skip('drive.downloadRange(dbRanges, blobRanges)', async (t) => {
-  const { drive, swarm, mirror, corestore } = await testenv(t)
+test('drive.downloadRange(dbRanges, blobRanges)', async (t) => {
```

Comment:
The previous test was testing something else entirely it seems. Your test makes more sense to me, but needs a good look from @mafintosh too because we're changing both the tests and the main logic, which is always dangerous.

Likewise for the other test